### PR TITLE
Add to officially using Keiko list

### PR DESCRIPTION
**Pull request by backjo**

### CODE REVIEW

**Comment by shrinandj** [comment]
👍 👍 👍

**Review by eytan-avisror — APPROVED** [review]
Thanks !

**Comment by eytan-avisror** [comment]
@backjo I think you just need to sign the commit

**Comment by backjo** [comment]
should be all set on the signature

File: README.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ Currently **officially** using Keiko:
 
 1. [Intuit](https://www.intuit.com/)
 1. [Zendesk](https://www.zendesk.com/)
+1. [LegalZoom](https://legalzoom.com)
```
Comment (eytan-avisror) [inline README.md:58]: Looks like Zendesk joined! can you fix the numbering?
Thanks!

Comment (eytan-avisror) [reply README.md:58]: Nevermind, I guess markdown displays this correctly even though it's all 1's 😆

**Pull request merged**